packages/db/tests/models/test-base-model.js

```diff
@@ -137,3 +146,37 @@ test('BasePgModel.exists() correctly returns false', async (t) => {
   const info = cryptoRandomString({ length: 5 });
   t.false(await basePgModel.exists(knex, { info }));
 });
+
+test('BasePgModel.delete() correctly deletes a record', async (t) => {
+  const { knex, basePgModel, tableName } = t.context;
+  const cumulusId = 123;
+
+  // Insert the record and validate that it exists in the table
+  await knex(tableName).insert({ cumulus_id: cumulusId });
+  t.like(
+    await basePgModel.get(knex, { cumulus_id: cumulusId }),
+    { cumulus_id: cumulusId }
+  );
+
+  // Delete the record and validate that it's gone
+  await basePgModel.delete(knex, { cumulus_id: cumulusId });
+  await t.throwsAsync(
+    basePgModel.get(knex, { cumulus_id: cumulusId }),
+    { instanceOf: RecordDoesNotExist }
+  );
+});
+
+test('BasePgModel.delete() works with knex transaction', async (t) => {
+  const { knex, basePgModel, tableName } = t.context;
+  const cumulusId = 789;
+  await knex(tableName).insert({ cumulus_id: cumulusId });
+  t.truthy(await knex.transaction(
```

Comment:
Yeah so `delete()` returns the output of `Knex.del()`, which is the number of rows affected. In this case it would just return `1` if it's successful. My thinking here was that we really don't care about the return value. Just that it's not an error and it's not `0` (which is falsey) but maybe that's not quite right. 

I could probably make this test pass by removing the `where` clause in the `delete()` method and have it delete all records in the table. That's not ideal 😄. 

Updating to check that only one record was deleted.